fedbiomed/node/environ.py

```diff
@@ -27,39 +27,34 @@
 from fedbiomed.common.constants import ComponentType, ErrorNumbers, HashingAlgorithms, DB_PREFIX, NODE_PREFIX
 from fedbiomed.common.environ import Environ
 from fedbiomed.transport.client import ResearcherCredentials
+from fedbiomed.common.config import NodeConfig
 
 
 class NodeEnviron(Environ):
 
     def __init__(self, root_dir: str = None):
         """Constructs NodeEnviron object """
         super().__init__(root_dir=root_dir)
-        logger.setLevel("INFO")
-        self._values["COMPONENT_TYPE"] = ComponentType.NODE
-        # Setup environment variables
-        self.setup_environment()
 
-    def default_config_file(self) -> str:
-        """Sets config file path """
+        self.config = NodeConfig(root_dir)
```

Comment:
Shouldn't `self.config` be a private `self._config` ? 
Same question for `fedbiomed.researcher.environ`

Reply:
Hi Marc,
`self.config` is the reference for configuration file which a static file. Environ is a dynamic object which may change based on env variable. I believe it is useful to make config non-private and accessible for other classes in case there is need to access original variable set in the config.

Reply:
Thank you Sergen,
By "access" do you mean read or read+modify :-) ?
- if read+modify this is fine to keep as it is
- if read only, then it would be nice to have private var + getter

Reply:
It is done